### ISSUE TITLE
Text commenting flow for a11y

### ISSUE THREAD

**Issue by kimisgold** [omeka/plugin-Commenting#55]
The `ajax` branch submits the comment form via ajax and more easily allows screen reader friendly notifications. Please test the following on an item show page:

* Comment on an item. Navigate to the commenting form via keyboard.
* Each field on the comment form should be properly labeled and read by the screen reader.
* When a comment is successfully submitted, the screen reader notifies the user.
* Submitted comments can be flagged for removal, as well as unflagged. The screen reader should announce the comment state when this happens.
* Users can reply to comments. Activating the reply link should place the comment form immediately under the reply target, and then focus on the first field of the commenting form.

**Comment by katknow** [comment]
This is working for the most part. The only unusual thing I'm finding is that NVDA (not JAWS or VoiceOver) is reading the text fields as ones that autocomplete across browsers, even though this does not seem to be true. Additionally, NVDA is referring to the comment area as an applicaton but not specifically a text field like JAWS or VoiceOver. I don't know if that is something fixable or if it is software dependent.

**Comment by kimisgold** [comment]
[This seems to be expected behavior for NVDA](https://github.com/nvaccess/nvda/issues/8633), because the browser can provide suggestions for those fields. The comment area is also considered an application because tinymce is acting a rich text editor. I think we're good here!
